dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogEndpoint.kt

```diff
@@ -6,78 +6,198 @@
 
 package com.datadog.android
 
-import com.datadog.android.Datadog.initialize
+import com.datadog.android.core.configuration.Configuration
 
 /**
  * This object contains constant values for all the Datadog Endpoint urls used in the SDK.
  */
 object DatadogEndpoint {
 
+    //  region Logs
+
+    /**
+     * The US1 endpoint for Logs (US based servers), used by default by the SDK.
+     * Use this in your [Configuration] if you log on [app.datadoghq.com](https://app.datadoghq.com)
+     * @see [Configuration]
+     */
+    const val LOGS_US1: String = "https://logs.browser-intake-datadoghq.com"
+
+    /**
+     * The US3 endpoint for Logs (US based servers).
+     * Use this in your [Configuration] if you log on [s3.datadoghq.com](https://us3.datadoghq.com)
+     * @see [Configuration]
+     */
+    const val LOGS_US3: String = "https://logs.browser-intake-us3-datadoghq.com"
+
+    /**
+     * The US1_FED endpoint for Logs (US based servers, FedRAMP compliant).
+     * Use this in your [Configuration] if you log on [app.ddog-gov.com](https://app.ddog-gov.com)
+     * @see [Configuration]
+     */
+    const val LOGS_US1_FED: String = "https://logs.browser-intake-ddog-gov.com"
+
+    /**
+     * The EU1 endpoint for Logs (EU based servers).
+     * Use this in your [Configuration] if you log on [app.datadoghq.eu](https://app.datadoghq.eu)
+     * @see [Configuration]
+     */
+    const val LOGS_EU1: String = "https://mobile-http-intake.logs.datadoghq.eu"
+
     /**
      * The endpoint for Logs (US based servers), used by default by the SDK.
-     * @see [DatadogConfig]
+     * @see [Configuration]
+     * @deprecated use [LOGS_US1] instead
      */
-    const val LOGS_US: String = "https://mobile-http-intake.logs.datadoghq.com"
```

Comment:
is this endpoint not being used anymore ? I see that you substitute with the `LOGS_US1` which has a different value now -> `https://logs.browser-intake-datadoghq.com`

Reply:
It's not the recommended one